product-service/serverless.yml

```diff
@@ -26,6 +26,35 @@ provider:
     PG_DATABASE: ${env:PG_DATABASE}
     PG_USERNAME: ${env:PG_USERNAME}
     PG_PASSWORD: ${env:PG_PASSWORD}
+    SNS_TOPIC_ARN: ${self:custom.SNS_TOPIC_ARN}
+
+  iamRoleStatements:
```

Comment:
I don't see any policy for SQS, it's stranges that it works :)

Reply:
I have it in `import-service/serverless.yml`
```
iamRoleStatements:
    - Effect: 'Allow'
      Action: 'sqs:*'
      Resource: ${self:custom.SQS_ARN}
```
Are you sure I need it in `product-service/serverless.yml`?
I don't call any SQS client functions in `product-service/handlers/catalogBatchProcess.ts`. Only consume SQSEvent.